app/src/main/java/ml/docilealligator/infinityforreddit/adapters/CommentsRecyclerViewAdapter.java

```diff
@@ -595,18 +595,17 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
                         ((LoadMoreChildCommentsViewHolder) holder).placeholderTextView.setText(R.string.loading);
 
                         Retrofit retrofit = mAccessToken == null ? mRetrofit : mOauthRetrofit;
+                        String sortType = mCommentRecyclerViewAdapterCallback.getSortType();
                         FetchComment.fetchMoreComment(mExecutor, new Handler(), retrofit, mAccessToken,
-                                parentComment.getMoreChildrenFullnames(),
-                                parentComment.getMoreChildrenStartingIndex(), parentComment.getDepth() + 1,
```

Comment:
This depth is `parentComment.getDepth() + 1`, not 0.

Reply:
Yes. But there are several reasons I did it this way:
- It works, at least for me. Unlike `/api/info` it seems `/api/morechildren` always sends comment depth, so it gets parsed fine
- Getting parent comment during parsing requires parsing current comment which requires depth which requires parent comment. Breaking the cycle is easy but because it works as is, I didn't want to add extra logic.
- Depth handling is already somewhat broken - on line 135 (old)/196 (new) depth is not incremented. It is no longer used to parse more children but I still wanted to point that out